packages/plugins/rudder_plugin_interface/lib/src/models/rudder_option.dart

```diff
@@ -3,17 +3,17 @@ import 'rudder_integration.dart';
 
 // we left fetching the external ids from the scratch here
 class RudderOption {
-  List<Map<String, dynamic>>? externalIds;
-  Map<String, Object>? integrations;
+  List<Map<String, String>>? externalIds;
+  Map<String, bool>? integrations;
```

Comment:
is thsi boolean or we can pass boolean and object for configuration of integration like in JS SDK?

Reply:
In Mobile, we do have the support for boolean only

Reply:
Now added support for sending object for configuration of integration like in JS SDK